tests/documentation/test_files/DocumentationStyleBear/bad_file4.py.test.correct

```diff
@@ -0,0 +1,12 @@
+def myfunc(a, b, x):
+    """
+
+    :param a:
+        slope
+    :param b:
+        offset
+    :param x:
+    :return:
+    """
+
```

Comment:
definitely.

just for idea: it's possible to use the range information of extracted comments to check for this^^
so we get the range and look into the next line